### PR TITLE
feat: add ports also to the container

### PR DESCRIPTION
**Pull request by smuu**
## Summary by CodeRabbit

- **New Features**
	- Enhanced network configuration capabilities for instances by adding TCP and UDP port specifications.
	- New functionality to define and expose network ports in container configurations.

- **Bug Fixes**
	- Improved error handling mechanisms to ensure accurate state checks during instance management.

- **Documentation**
	- Updated method signatures and struct definitions to reflect the new network port fields.

### CODE REVIEW

**Review by MSevey — DISMISSED** [review]
Tests?

File: pkg/instance/execution.go [hunk 1]
```diff
@@ -384,6 +384,8 @@ func (e *execution) prepareReplicaSetConfig() k8s.ReplicaSetConfig {
 		StartupProbe:    e.instance.monitoring.startupProbe,
 		Files:           e.instance.storage.files,
 		SecurityContext: e.instance.security.prepareSecurityContext(),
+		TCPPorts:        e.instance.network.portsTCP,
+		UDPPorts:        e.instance.network.portsUDP,
```
[the remark this answers is not in the document]
Comment (smuu) [reply pkg/instance/execution.go:388]: thanks, added them

**Comment by smuu** [comment]
> Tests?

Tests are failing in the netshaper, which has nothing todo with this change.
The netshaper test is flaky for a long time.

As this feature is not needed anymore, I am considering removing it.

**Review by mojtaba-esk — APPROVED** [review]
Nice work!

**Pull request merged**